packages/astro/src/core/build/plugins/plugin-ssr.ts

```diff
@@ -178,13 +179,11 @@ function generateSSRCode(adapter: AstroAdapter) {
 		`const _manifest = Object.assign(defaultManifest, {`,
 		`    pageMap,`,
 		`    serverIslandMap,`,
-		`    renderers,`,
-		`    actions: () => import("${ENTRYPOINT_VIRTUAL_MODULE_ID}"),`,
```

Comment:
Why are these removed? I may know why, and if so, we should remove `middleware` too.

Reply:
The manifest extension happens here now: https://github.com/withastro/astro/blob/feat/environment-api/packages/astro/src/core/app/entrypoint.ts

Reply:
Actually, that's a bug we need to fix, can you track it? Middleware shouldn't be added in `App` if edge middleware is enabled, but at the moment we don't do that